ruleset/sca/windows/cis_win10_enterprise.yml

```
@@ -942,30 +986,34 @@ checks:
     remediation: "To establish the recommended configuration via GP, set the following UI path to Enabled: Computer Configuration\\Policies\\Windows Settings\\Security Settings\\Local Policies\\Security Options\\Network security: Do not store LAN Manager hash value on next password change."
     compliance:
       - cis: ["2.3.11.5"]
-      - cis_csc: ["16.14"]
+      - cis_csc: ["3.11"]
       - pci_dss: ["8.2.1"]
       - tsc: ["CC6.1"]
     references:
-      - 'CCE-36326-7'
-    condition: all
+      - "CCE-36326-7"
+    condition: any
     rules:
+      - 'r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa'
```

Using condition 'any' probably means this should be:
```suggestion
      - 'not r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa'
```

---

Done 18dbade